### PR TITLE
feat: add line item property dtos

### PR DESCRIPTION
**Pull request by connor-mohc**

### CODE REVIEW

File: properties.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		properties = append(properties, dto.ToShopify())
+	}
+
+	return properties
+}
+
+// PropertyDTO holds custom information for an entity.
+type PropertyDTO struct {
+	// Name is the name of the property.
+	Name string `json:"name"`
```
Comment (Ianmohc) [inline properties.go:21]: This formatted... looks a little weird.

**Review by Ianmohc — CHANGES_REQUESTED** [review]

File: properties.go (hunk 1 above)
Comment (connor-mohc) [reply properties.go:21]: Must be GitHub, it's the correct spacing :thinking:

Comment (connor-mohc) [reply properties.go]: So dto properties don't usually have comments on so removed them, aligns fine now.

Comment (Ianmohc) [reply properties.go]: Oh that was why then.

Comment (Ianmohc) [reply properties.go]: Probably does not need them anyway. Obvious names.

**Pull request merged**